app/views/docs/migrations.phtml

```diff
@@ -0,0 +1,86 @@
+            <td><span class="tag green">supported</span></td>
+            <td><span class="tag yellow">partial</span></td>
+            <td><span class="tag green">supported</span></td>
+            <td><span class="tag">manual</span></td>
+        </tr>
+        <tr>
+            <td>
+                <p><a href="/docs/migrations-supabase"><i class="icon-angle-circled-right margin-start-negative-tiny margin-end-tiny"></i> Supabase</a></p>
+            </td>
+            <td><span class="tag green">supported</span></td>
+            <td><span class="tag green">supported</span></td>
+            <td><span class="tag yellow">partial</span></td>
+            <td><span class="tag green">supported</span></td>
+            <td><span class="tag">manual</span></td>
+        </tr> 
+        <tr>
+            <td>
+                <p><a href="/docs/migrations-nhost"><i class="icon-angle-circled-right margin-start-negative-tiny margin-end-tiny"></i> NHost</a></p>
+            </td>
+            <td><span class="tag green">supported</span></td>
+            <td><span class="tag green">supported</span></td>
+            <td><span class="tag yellow">partial</span></td>
+            <td><span class="tag green">supported</span></td>
+            <td><span class="tag">manual</span></td>
+        </tr>   
+        <tr>
+            <td>
+                <p><a href="/docs/migrations-cloud-to-local"><i class="icon-angle-circled-right margin-start-negative-tiny margin-end-tiny"></i> Cloud to Self-hosted</a></p>
+            </td>
+            <td><span class="tag green">supported</span></td>
+            <td><span class="tag green">supported</span></td>
+            <td><span class="tag green">supported</span></td>
+            <td><span class="tag green">supported</span></td>
+            <td><span class="tag green">supported</span></td>
+        </tr>   
+        <tr>
+            <td>
+                <p><a href="/docs/migrations-local-to-cloud"><i class="icon-angle-circled-right margin-start-negative-tiny margin-end-tiny"></i> Self-hosted to Cloud</a></p>
+            </td>
+            <td><span class="tag green">supported</span></td>
+            <td><span class="tag green">supported</span></td>
+            <td><span class="tag green">supported</span></td>
+            <td><span class="tag green">supported</span></td>
+            <td><span class="tag green">supported</span></td>
+        </tr>         
+    </tbody>
+</table>
+
+<h2><a href="#limitations" id="limitations">Limitations</a></h2>
+<p>
+    Migrations cannot transfer all data perfectly, so certain fields, such as <code>$createdAt</code> and <code>$updatedAt</code>, may not be transferred. 
+    More information can be found on the migration page for each source.
+</p>
+
+<p>
+    Migrations help you jump start your move, but because each product is unique, complex databases and product unique features like functions need to be migrated manually.
```

Comment:
```suggestion
    Migrations help you jump start your move, but because each product is unique, complex databases and product unique features like functions or permissions might need to be migrated manually.
```